patterns/testimonial.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Title: Testimonial
+ * Slug: twentytwentyfour/testimonial
+ * Categories: testimonials
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"140px","bottom":"140px"}}},"backgroundColor":"contrast","textColor":"tertiary","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-tertiary-color has-contrast-background-color has-text-color has-background" style="padding-top:140px;padding-bottom:140px"><!-- wp:paragraph {"align":"center","style":{"spacing":{"margin":{"bottom":"32px"}},"typography":{"lineHeight":"1.2"}},"textColor":"tertiary","fontSize":"x-large","fontFamily":"cardo"} -->
```

Comment:
Let's use rems instead of px for the margins and paddings all over the pattern

Reply:
Will do! Thanks